rnlp/parse.py

```diff
@@ -149,7 +152,18 @@ def makeIdentifiers(blocks, target="sentenceContainsTarget(+SID,+WID).",
                     makeIdentifiers(blocks)
                     # 100%|██████████████████████| 2/2 [00:00<00:00, 18.49it/s]
     """
-
+    # Specify a global variable
+    global OUTPUT_DIR
+    if output is None:
+        # If no output passed in, set to the current directory
+        OUTPUT_DIR = os.getcwd()
+    else:
+        # Set to what was passed in
+        OUTPUT_DIR = output
+        # Ensure it exists
+        if not os.path.isdir(OUTPUT_DIR):
+            os.mkdir(OUTPUT_DIR)
```

Comment:
We should replace `os.mkdir` with [`os.makedirs`](https://docs.python.org/3/library/os.html#os.makedirs):

```python
if not os.path.isdir(output_dir):
    os.makedirs(output_dir, exist_ok=True)
```

`os.makedirs` should avoid problems that would otherwise be raised with nested directories: `/path/to/output/`